xml2rfc_compat/serializer.py

```diff
@@ -2,8 +2,8 @@
 Registers :func:`relaton.serializers.bibxml_string.serialize`
```

Comment:
@stefanomunarini this comment should be updated too, since we don’t have `relaton.serializers.bibxml_string` anymore (we’re registering a function defined in this same module now)